Procfile

```diff
@@ -0,0 +1 @@
+web: bin/boot
```

Comment:
I switched over to a letsencrypt cert and got away with out using the Procfile somehow. https://github.com/atmos/atmos.github.io/pull/9